docs/en/sql-reference/aggregate-functions/combinators.md

```diff
@@ -51,33 +51,111 @@ Converts an aggregate function for tables into an aggregate function for arrays
 
 ## -OrDefault {#agg-functions-combinator-ordefault}
 
-Fills the default value of the aggregate function’s return type if there is nothing to aggregate.
+Changes behavior of an aggregate function.
+
+If an aggregate function doesn't have input values, with this combinator it returns the default value for its return data type. Applies to the aggregate functions that can take empty input data.
+
+`-OrDefault` can be used with other combinators.
+
+**Syntax** 
+
+``` sql
+<aggFunction>OrDefault(x)
+```
+
+**Parameters**
+
+- `x` — Aggregate function parameters.
+
+**Returned values** 
+ 
+Returns the default value of an aggregate function’s return type if there is nothing to aggregate.
+
+Type depends on the aggregate function used.
+
+**Example**
+
+Query:
 
 ``` sql
 SELECT avg(number), avgOrDefault(number) FROM numbers(0)
 ```
 
+Result:
+
 ``` text
 ┌─avg(number)─┬─avgOrDefault(number)─┐
 │         nan │                    0 │
 └─────────────┴──────────────────────┘
 ```
 
+Also `-OrDefault` can be used with another combinators. It is useful when the aggregate function does not accept the empty input.
```

Comment:
Not for this review, but probably we need some explicit part about combinators combinations.
- precedence of operators if any
- is there only one way to write such aggregation name or it allows all variants avgOrDefaultIf/avgIfOrDefault
- which combinators can not be combined
- whether result changes if we change way we wrote aggregation name  avgOrDefaultIf<->avgIfOrDefault.
- do we apply functions from left to right or not